exporter/otlp-http/README.md

```diff
@@ -0,0 +1,102 @@
+# opentelemetry-exporter-otlp-http
+
+The `opentelemetry-exporter-otlp-http` gem provides an [OTLP](https://github.com/open-telemetry/opentelemetry-proto) over http exporter for OpenTelemetry for Ruby. Using `opentelemetry-exporter-otlp-http`, an application can configure OpenTelemetry to export collected tracing data to [the OpenTelemetry Collector][opentelemetry-collector-home].
+
+## What is OpenTelemetry?
+
+[OpenTelemetry][opentelemetry-home] is an open source observability framework, providing a general-purpose API, SDK, and related tools required for the instrumentation of cloud-native software, frameworks, and libraries.
+
+OpenTelemetry provides a single set of APIs, libraries, agents, and collector services to capture distributed traces and metrics from your application. You can analyze them using Prometheus, Jaeger, and other observability tools.
+
+## How does this gem fit in?
+
+The `opentelemetry-exporter-otlp-http` gem is a plugin that provides OTLP export. To export to the OpenTelemetry Collector, an application can include this gem along with `opentelemetry-sdk`, and configure the `SDK` to use the provided OTLP exporter as a span processor.
+
+Generally, *libraries* that produce telemetry data should avoid depending directly on specific exporter, deferring that choice to the application developer.
+
+### Supported protocol version
+
+This gem supports the [v0.11.0 release](https://github.com/open-telemetry/opentelemetry-proto/releases/tag/v0.11.0) of OTLP.
+
+## How do I get started?
+
+Install the gem using:
+
+```
+gem install opentelemetry-sdk
+gem install opentelemetry-exporter-otlp-http
+```
+
+Or, if you use [bundler][bundler-home], include `opentelemetry-sdk` in your `Gemfile`.
+
+Then, configure the SDK to use the OTLP exporter as a span processor, and use the OpenTelemetry interfaces to produces traces and other information. Following is a basic example.
+
+```ruby
+require 'opentelemetry/sdk'
+require 'opentelemetry/exporter/otlp'
```

Comment:
Does this still work or should it be:
```suggestion
require 'opentelemetry/exporter/otlp/http'
```
I suppose the next question would be: if the old example doesn't work anymore, that seems like a breaking change - do we want that or should we make the old way work (but still fix the example)?

Reply:
Ignore the second part of my comment. I think the suggestion is relevant, tho